main.go

```diff
@@ -196,7 +248,25 @@ func connectAction(ctx *cli.Context) error {
 	}
 	durations["rhsm"] = time.Since(start)
 
-	/* 2. Register insights-client */
+	/* 2 Show Profile information */
+	// Generate a new http client with mtls configure
+	client := http.NewHTTPClient()
+	if client != nil {
+		// Get the user profile
+		profile, err := getUserProfile(client)
+		if err != nil {
+			errorMessages["rhsm"] = fmt.Errorf("Cannot get the user profile: %w", err)
```

Comment:
"user profile" is misleading. It's not a profile of user settings, it's a profile of configured state. "system profile" or "configuration profile" is more accurate.